umh.docs.umh.app/content/en/docs/production-guide/installation/advanced_installation.md

```
@@ -0,0 +1,231 @@
+
+<!-- overview -->
+
+This page describes advanced requirements and considerations when installing the United Manufacturing Hub.
+It is meant to be an additinal guide for the [Installation guide](/docs/getstarted/installation) page.
+
+## Storage Configuration
+
+When setting up your virtual machine for UMH:
+
+**Partition Layout**
+   - The default Rocky Linux installation typically creates a large `/home` partition
```

Markdownlint (0.37.0)

Expected: 0; Actual: 3
Unordered list indentation

(MD007, ul-indent)